<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameWindow</class>
 <widget class="QWidget" name="NewGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
        background-color: #3e4e3e;
        border: 2px solid #6a7a6a;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    QPushButton {
        background-color: #7e8c72;
        border-top: 2px solid #d7dacd;
        border-left: 2px solid #d7dacd;
        border-bottom: 2px solid #4a4a4a;
        border-right: 2px solid #4a4a4a;
        padding: 4px 10px;
        color: white;
        font-weight: bold;
		 font-size: 10pt;
    }

    QPushButton:hover {
        background-color: #96a783;
    }

    QPushButton:pressed {
        border-top: 2px solid #4a4a4a;
        border-left: 2px solid #4a4a4a;
        border-bottom: 2px solid #d7dacd;
        border-right: 2px solid #d7dacd;
        background-color: #6b7960;
    }</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font-size: 14pt;
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font-size: 14pt;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 15pt;
	border: &quot;none&quot;;
    color: #d3b84b;
	font-family: &quot;Verdana&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Create Game</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: #d3b84b;
border: &quot;none&quot;;
	font-family: &quot;Verdana&quot;;
}</string>
       </property>
       <property name="text">
        <string>Player Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #4a5236;  
    color: #d3b84b;             
    border: 1px solid #606640;   
    padding: 5px 18px 5px 5px;  
    border-radius: 3px;          
    min-width: 6em;
}

QLineEdit:hover {
    border: 1px solid #a4b066; 
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: #d3b84b;
border: &quot;none&quot;;
	font-family: &quot;Verdana&quot;;
}</string>
       </property>
       <property name="text">
        <string>Game Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #4a5236;  
    color: #d3b84b;             
    border: 1px solid #606640;   
    padding: 5px 18px 5px 5px;  
    border-radius: 3px;          
    min-width: 6em;
}

QLineEdit:hover {
    border: 1px solid #a4b066; 
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: #d3b84b;
border: &quot;none&quot;;
	font-family: &quot;Verdana&quot;;
}</string>
       </property>
       <property name="text">
        <string>Players</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="playersSpinBox">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
    background-color: #4a5236;  
    color: #d3b84b;             
    border: 1px solid #606640;   
    padding: 5px 18px 5px 5px;;
    border-radius: 3px;          
}

QSpinBox:hover {
    border: 1px solid #a4b066; 
}

QComboBox QAbstractItemView {
    background-color: #4a5236;
    border: 1px solid #606640;
    selection-background-color: #636e3f;
    color: #d3b84b;
}

QSpinBox::focus {
    selection-background-color: transparent;
    selection-color: #d3b84b;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: #d3b84b;
border: &quot;none&quot;;
	font-family: &quot;Verdana&quot;;
}</string>
       </property>
       <property name="text">
        <string>Rounds</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="roundsSpinBox">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
    background-color: #4a5236;  
    color: #d3b84b;             
    border: 1px solid #606640;   
    padding: 5px 18px 5px 5px;;
    border-radius: 3px;          
}

QSpinBox:hover {
    border: 1px solid #a4b066; 
}

QComboBox QAbstractItemView {
    background-color: #4a5236;
    border: 1px solid #606640;
    selection-background-color: #636e3f;
    color: #d3b84b;
}

QSpinBox::focus {
    selection-background-color: transparent;
    selection-color: #d3b84b;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="mapList">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
border: none;
}

QListWidget::item {
    border: 2px solid transparent;
    padding: 5px;
	background: none;
}
QListWidget::item:selected {
    border: 2px solid #bfa949;
	border-radius: 0;
}</string>
    </property>
    <property name="movement">
     <enum>QListView::Movement::Static</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::ResizeMode::Adjust</enum>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="viewMode">
     <enum>QListView::ViewMode::IconMode</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignmentFlag::AlignHCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
